clang/lib/AST/Expr.cpp

```diff
@@ -503,6 +503,68 @@ SourceLocation DeclRefExpr::getEndLoc() const {
   return getNameInfo().getEndLoc();
 }
 
+UniqueStableNameExpr::UniqueStableNameExpr(SourceLocation OpLoc,
+                                           SourceLocation LParen,
+                                           SourceLocation RParen,
+                                           QualType ResultTy,
+                                           TypeSourceInfo *TSI)
+    : Expr(UniqueStableNameExprClass, ResultTy, VK_LValue, OK_Ordinary),
```

Comment:
I think you're right.  The last time I implemented this I left it as an LValue (since it was a PredefinedExpr) and just copy/pasted out of this.  When I re-inserted that yesterday I thought it was particularly suspicious but hoped it would show up in review/tests.  I shoulda put a TODO there.